.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -136,6 +137,7 @@ jobs:
             $LLVM_LAZY_JIT_BUILD_OPTIONS,
             $LLVM_EAGER_JIT_BUILD_OPTIONS,
             $MULTI_TIER_JIT_BUILD_OPTIONS,
+            $SHARED_BUILD_OPTIONS,
```

Comment:
not here. this is for variant running mode only.

Reply:
You might want to add `SHARED_BUILD_OPTIONS` into `make_options_feature`. It will test `SHARED_BUILD_OPTIONS` with all kinds of running mode

Reply:
The change made at line 139 should be undone. That section is intended solely for the run mode configuration.